kpc/views.py

```diff
@@ -138,21 +141,40 @@ def test_func(self):
 
 
 class CertificateView(BaseCertificateView):
+    REVIEW_MSG = "Please review the certificate data below."
```

Comment:
Curious should this message only be applicable to the `CertificateConfirmView`? If so, should the logic in `get_form` be moved?